### PR TITLE
Filling the description for the 3rd section

### PR DESCRIPTION
**Pull request by WojcikMike**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,12 +22,27 @@
...
+Looking at the web server logs to look failures related to http requests that were doing updates. Any 4xx or 5xx errors related to PUT, POST, DELETE requests indicate that probably such issue occurred in the application.
+ 
 * Not all failures are visible in the server logs => timeout on the way back to the client
+Not every failure will be visible in the web server logs. Lost responses from web server to the client will not be logged. Lack of errors in the log does not mean that such problems never took place. 
+
 * Do you even have logs? are you checking them?
+Logs need to be configured and looked at regular basis. Certain web servers like IIS have logging configured but using console applications to host your web api might not. 
+
 * Catching and logging http related exceptions in the client
+Most of the exceptions  will be visible on the client side, and should be caught and logged. ???
```
Comment (SzymonPobiega) [inline README.md]: Not sure I understand?

Comment (WojcikMike) [reply README.md]: I am not sure what to put here beyond the sentence that I wrote

Comment (kbaley) [reply README.md]: Do you mean client-side exceptions should be logged on the server? Or server-side exceptions should be displayed on the client?

File: README.md [hunk 2]
```diff
@@ -22,12 +22,27 @@
 ### Ok, I get that this is a problem, how to find out that I'm affected?
   
 * Failures in the server log of http requests that are making updates 
+Looking at the web server logs to look failures related to http requests that were doing updates. Any 4xx or 5xx errors related to PUT, POST, DELETE requests indicate that probably such issue occurred in the application.
```
Comment (andreasohlund) [inline README.md]: >Looking at the web server logs to look failures related to http requests that were doing updates

How about

Monitoring web server logs for failures related to http requests modifying data

File: README.md (hunk 1 above)
Comment (andreasohlund) [inline README.md]: Isn't this the continuation of the first sentence, ie not all errors will be in the server logs so to have "full coverage" you need to catch http releated errors in the client as well?

So perhaps remove this and extend the first sentence?

File: README.md [hunk 3]
```diff
@@ -22,12 +22,27 @@
...
+
 * Happens to all types of web apps , SPA's, webform, mvc
-* The client might not be owned by you, eg. API gateway for B2B
-all the messy support interactions to deal with that
+
+This issues happen in every application that uses http requests like web apps, SPA's, mVC but also when calling web api etc.
+
+* The client might not be owned by you, eg. API gateway for B2B all the messy support interactions to deal with 
+
+The same issue exists in integration scenarios when a client is not owned by you, and simply calling http api.
```
Comment (andreasohlund) [inline README.md]: ... which makes it more complex given that you don't own the client?

**Review by andreasohlund — CHANGES_REQUESTED** [review]

**Review by andreasohlund — APPROVED** [review]

**Comment by andreasohlund** [comment]
@SzymonPobiega can you take a quick glance and merge?

**Review by SzymonPobiega — APPROVED** [review]

**Pull request merged**